src-docs/src/views/theme/other/_levels_js.tsx

```diff
@@ -18,11 +23,114 @@ export default () => {
 
   return (
     <>
+      <ThemeExample
+        title="euiTheme.levels[type]"
+        description={
+          <>
+            <p>
+              Most of the time the z-index is handled per component. But you can
+              use the table below as a reference to how EUI layers the
+              components.
+            </p>
+          </>
+        }
+        examplePanel={{
+          color: 'subdued',
+        }}
+        example={
+          <EuiPanel
+            css={css`
+              width: 100%;
+              height: 200px;
+              position: relative;
+            `}
+          >
+            <EuiPanel
+              color="primary"
+              paddingSize="none"
+              css={css`
+                width: 40%;
+                height: 80%;
+                position: absolute;
+                left: 30%;
+                top: 10%;
+                display: flex;
+                align-items: center;
+                justify-content: center;
+                z-index: ${euiTheme.levels.modal};
+                ${useEuiShadow('s', euiTheme.colors.primary)};
+              `}
+            >
+              <EuiText size="xs">
+                <p>
+                  <strong>Modal</strong>
+                </p>
+              </EuiText>
+            </EuiPanel>
+            <EuiPanel
+              color="danger"
+              paddingSize="none"
+              css={css`
+                width: 40%;
+                height: 30%;
+                position: absolute;
+                right: 2%;
+                top: 2%;
+                display: flex;
+                align-items: center;
+                justify-content: center;
+                z-index: ${euiTheme.levels.toast};
+                ${useEuiShadow('s', euiTheme.colors.danger)};
```

Comment:
This is a cute idea to make the shadow the same color, but to me it makes me think my eyes cant focus because the edges are now blurry.